lib/compilers/mojo.ts

```diff
@@ -0,0 +1,59 @@
+import path from 'node:path';
+import type {ParseFiltersAndOutputOptions} from '../../types/features/filters.interfaces.js';
+// Copyright (c) 2019, Sebastian Rath
```

Comment:
Could you move the imports below the license clause? And is the license name+year correct?